test/proxy.spec.js

```diff
@@ -82,6 +82,12 @@ it('should exclude patterns', async ({browserType, defaultBrowserOptions, server
     expect(error.message).toBeTruthy();
   }
 
+  if (CHROMIUM) {
+    // Should successfully navigate to the error page.
+    await page.waitForEvent('framenavigated', frame => frame.url() === 'chrome-error://chromewebdata/');
```

Comment:
Couldn't all of the `framenavigated` events fire before `page.goto` resolves? Then this would time out right?

Reply:
I expect page.goto to fail after request failed, and only after that error page should commit. Let's see on the bots.